client/signup/steps/domains/index.jsx

```diff
@@ -88,7 +98,7 @@ module.exports = React.createClass( {
 				} ) :
 				undefined;
 
-		SignupActions.submitSignupStep( {
+		SignupActions.submitSignupStep( Object.assign( {}, {
```

Comment:
This is super minor and doesn't need to be changed, but I don't think you need to worry about mutating an object literal here, and could probably remove the first argument. Please correct me if I'm wrong about this.